monkeytype/util.py

```diff
@@ -35,17 +40,19 @@ def get_func_in_module(module: str, qualname: str) -> Callable:
     # https://github.com/python/typeshed/blob/master/stdlib/3/inspect.pyi#L213
     func = inspect.unwrap(func)  # type: ignore
     if isinstance(func, types.MethodType):
-        func = func.__func__
+        return func.__func__
```

Comment:
Why this change (and below)?

Reply:
Personally, I think it's a lot easier to trace. Once we pass certain conditions there's really no point in continuing down into the rest of the function if we can just exit out early with a known result. I think it could also prevent future people from altering `func` later if we explicitly declare the exact object we want to return with an early return statement.

I'm perfectly okay reverting this change if you disagree.

Reply:
Let's revert, it's not related to the changes in this diff. It's a judgment call whether later additions to this function are more likely to be things that should or shouldn't also apply to these unwrapped values, I can envision either possibility. If they shouldn't apply, the existing `elif` chain pattern handles that fine. I prefer to err on the side of single return per function, early returns are easy to miss.